forest/src/cli/config.rs

```diff
@@ -5,18 +5,30 @@ use forest_chain_sync::SyncConfig;
 use forest_libp2p::Libp2pConfig;
 use forest_networks::ChainConfig;
 use serde::{Deserialize, Serialize};
-use std::sync::Arc;
+use std::{collections::HashSet, sync::Arc};
 
 use super::client::Client;
 
-#[derive(Serialize, Deserialize, PartialEq, Default)]
+#[derive(Serialize, Deserialize, Default, Clone, PartialEq)]
+pub struct LogConfig {
+    pub log_values: HashSet<LogValue>,
+}
+
+#[derive(Serialize, Deserialize, PartialEq, Eq, Hash, Clone)]
+pub struct LogValue {
+    pub module: String,
+    pub level: String,
```

Comment:
For one given module, is it possible to have multiple levels? Like `warn` and `error`?

Reply:
This isn't needed as upper level will take lower levels by default